Formula/libdatrie.rb

```diff
@@ -0,0 +1,36 @@
+class Libdatrie < Formula
+  desc "Double-Array Trie Library"
+  homepage "https://linux.thai.net/~thep/datrie/datrie.html"
+  url "https://github.com/tlwg/libdatrie/releases/download/v0.2.13/libdatrie-0.2.13.tar.xz"
+  sha256 "12231bb2be2581a7f0fb9904092d24b0ed2a271a16835071ed97bed65267f4be"
+  license "LGPL-2.1-or-later"
+
+  def install
+    ENV.deparallelize
```

Comment:
install fails in parallel mode:
```
libtool: link: clang -g -O2 -o .libs/trietool trietool.o  ../datrie/.libs/libdatrie.dylib -liconv
/Library/Developer/CommandLineTools/usr/bin/make  install-data-hook
 ../build-aux/install-sh -c -d '/opt/homebrew/Cellar/libdatrie/0.2.13/bin'
rm -f /opt/homebrew/Cellar/libdatrie/0.2.13/bin/trietool-0.2
ln -s trietool /opt/homebrew/Cellar/libdatrie/0.2.13/bin/trietool-0.2
ln: /opt/homebrew/Cellar/libdatrie/0.2.13/bin/trietool-0.2: No such file or directory
make[3]: *** [install-data-hook] Error 1
```

Reply:
Could you report this upstream too?

Reply:
I'm honestly not too sure what exactly is going wrong there as a manual install succeeded

Reply:
Is your manual install parallelised?